frontends/concrete-python/Makefile

```diff
@@ -75,7 +75,7 @@ pytest-default: tfhers-utils
 pytest-macos:
 	pytest tests -svv -n auto \
 		--key-cache "${KEY_CACHE_DIRECTORY}" \
-		-m "${PYTEST_MARKERS} minimal"
```

Comment:
what were markers previously passed to this?

Reply:
not dataflow and not graphivz